plugins/woocommerce-admin/client/marketplace/components/product-list/product-list.tsx

```diff
@@ -1,7 +1,3 @@
-/**
```

Comment:
I'm so confused - I thought removing these was generating linting errors, even if they're empty. It doesn't seem to now 😕

Reply:
Yup, confirming I'm not seeing any errors locally.